d2bs/kolbot/libs/bots/MFHelper.js

```diff
@@ -236,5 +237,6 @@ MainLoop:
 		delay(100);
 	}
 
+	Config.MFHelper.BreakClearLevel = false; // revert to default value
```

Comment:
You cant assume this is the value that was set

Reply:
even @kolton used the overwrite of Config values - https://github.com/blizzhackers/kolbot/blob/master/d2bs/kolbot/libs/bots/Follower.js#L548-L551